tests/js/lib/instrument-source.js

```diff
@@ -0,0 +1,51 @@
+/*
+
+    Instrument our source including config files and templates for use in preparing browser coverage reports
+    with gpii-webdriver/.
+
+ */
+/* eslint-env node */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-testem");
+require("../../../");
+
+fluid.registerNamespace("gpii.tests.handlebars.instrumentation");
+
+gpii.tests.handlebars.instrumentation.options = {
+    sources:    ["./src/**/*.js", "./index.js"],
+    excludes:   [],
+    nonSources: [
+        "./tests/**",
+        "./Gruntfile.js",
+        "./node_modules/gpii-binder/src/js/binder.js",
```

Comment:
Why does anything from within our top-level node_modules explicitly need to be excluded?

Reply:
That's not an exclusion.  Each entry is a "non source" file that should not be instrumented, but that still needs to be served up by the express instance we use with Testem.